packages/dialect-react-ui/components/Chat/screens/Error.tsx

```diff
@@ -1,4 +1,3 @@
-import React from 'react';
```

Comment:
shouldn't it cause problems after transpilation?

Reply:
nope, since tsconfig is set to `react-jsx`, a new import gets added: `import { jsx as _jsx } from 'react/jsx-runtime'`

Reply:
coool